graphql/resolvers.go

```diff
@@ -2871,13 +2871,26 @@ func (r *taskResolver) IsPerfPluginEnabled(ctx context.Context, obj *restModel.A
 				return false, err
 			}
 			for _, projectName := range perfPlugin.Projects {
-				if projectName == pRef.Id || projectName == pRef.Identifier {
-					return true, nil
+				if projectName != pRef.Id || projectName != pRef.Identifier {
+					return false, nil
 				}
 			}
 		}
+		opts := apimodels.GetCedarPerfCountOptions{
+			BaseURL: evergreen.GetEnvironment().Settings().Cedar.BaseURL,
+			TaskID:  *obj.Id,
+		}
+
+		result, err := apimodels.CedarPerfResultsCount(ctx, opts)
+		if err != nil {
+			return false, err
```

Comment:
This should be returning a graphql error. `InternalServerError.Send(ctx, fmt.Sprintf("error requesting perf data from cedar: ", err))`